services/repository-managers/pom.xml

```diff
@@ -58,7 +59,17 @@
 			<groupId>org.sagebionetworks</groupId>
 			<artifactId>simple-http-client</artifactId>
 		</dependency>
-
+		<!-- Force maven to use our modified version of json.org by including it 
+			before com.github.everit-org.json-schema which also depends on it. -->
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>JSON-Java</artifactId>
+		</dependency>
+		<!-- JSON schema validation library -->
+		<dependency>
+			<groupId>com.github.everit-org.json-schema</groupId>
+			<artifactId>org.everit.json.schema</artifactId>

```

Comment:
Please add the exclusion block on the conflicting org.json dependency so we are sure that it's not added to the classpath.

Reply:
This might potentially be added to the root pom.xml in the dependency management section